rasa/core/policies/ted_policy.py

```diff
@@ -22,13 +22,15 @@
 from rasa.shared.nlu.interpreter import NaturalLanguageInterpreter
 from rasa.core.policies.policy import Policy, PolicyPrediction
 from rasa.core.constants import DEFAULT_POLICY_PRIORITY, DIALOGUE
+from rasa.shared.constants import DIAGNOSTIC_DATA
 from rasa.shared.core.constants import ACTIVE_LOOP, SLOTS
 from rasa.shared.core.trackers import DialogueStateTracker
 from rasa.shared.core.generator import TrackerWithCachedStates
 from rasa.utils import train_utils
 from rasa.utils.tensorflow.models import RasaModel, TransformerRasaModel
 from rasa.utils.tensorflow.model_data import RasaModelData, FeatureSignature
 from rasa.utils.tensorflow.model_data_utils import convert_to_data_format
+from rasa.shared.utils.tensorflow.tf_to_numpy import values_to_numpy
```

Comment:
@wochinge this import is not according to our guidelines, right?

Reply:
I have to put this into the shared package because it is used by `Message`.

Reply:
I mean I remember there was a discussion that we want to use the full path for functions, but then the function call will be longer than 2 lines or smth 🙈

Reply:
You should at least reference the function by module @JEM-Mosig

Reply:
It is no longer in `shared`, so this is obsolete.